### PR TITLE
feat: git tutorials page

### LINKED ISSUE #45: Git tutorials to MDX
Convert Git tutorials to a page on the website

### PR DESCRIPTION
**Pull request by exu3**
This PR converts git.md to mdx and adds an author section to the layout component

closes #45

### CODE REVIEW

**Comment by awlevin** [comment]
#### 📝Changed routes:


| [**/about**](https://open-ipll7sbk3-thinkfiveable.vercel.app/about) | [**/docs/git**](https://open-ipll7sbk3-thinkfiveable.vercel.app/docs/git) |
|:-:|:-:|
|<a href="https://open-ipll7sbk3-thinkfiveable.vercel.app/about">Screenshot of /about</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/4vk-w3oO7YaXbOqCu77jgZ5mhDsUVa6Lmn_7OyCxOGZg-4gjIS6ew-WnndOvYIOD8FNqdYxNwhmIQfC1e9Uevg.png">(view full size)</a> |<a href="https://open-ipll7sbk3-thinkfiveable.vercel.app/docs/git">Screenshot of /docs/git</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/5VEzEoHgEMaC5ZId-rVw36Gz3SFgHcV_i3o5JDVml0cyZZA4ImAs8TGY3qfQG1BLEJqYb7eE4bwgtYKgJailtA.png">(view full size)</a> |







Commit 6deb61f2c8423b657256c004b2b4cc9943f33d55 (https://open-ipll7sbk3-thinkfiveable.vercel.app).

**Comment by zaida04** [comment]
I don't see any buttons on the page that direct users to the rendered mdx page. Is there supposed to be a button somewhere?

**Comment by exu3** [comment]
> I don't see any buttons on the page that direct users to the rendered mdx page. Is there supposed to be a button somewhere?

@zaida04 currently, there isn't a button linking to the page, but I can link the Docs button in the footer to this page for now?

**Review by sarthaktexas — DISMISSED** [review]

**Comment by awlevin** [comment]
I wonder if there’s a good spot for another button near “learn more” like “getting started” or something? This is an awesome PR and I’d love for it to have higher visibility if that makes sense to y’all too.

**Comment by zaida04** [comment]
In addition to ^, the text leading to the git tutorials isn’t bolded, and is there some way we can include you in the author section too? You helped to write the PR, so it’s only fair you receive recognition for it as well.

File: components/layout.js [hunk 1]
```diff
@@ -16,8 +16,21 @@ export default function Layout({ subtitle, title, children }) {
...
+                {author && (
+                    <div className="flex flex-row align-middle space-x-3">
+                        <img src={authorPfp} alt={authorName} className="w-14 h-14 rounded-full" />
+                        <div className="-space-y-1">
+                            <p className="font-mono font-medium text-2xl">{authorName}</p>
+                            <p className="font-rubik font-light text-lg">{publishDate}</p>
+                        </div>
+                    </div>
+                )}
+                {/* <div className="flex flex-row">
```
Comment (sarthaktexas) [inline components/layout.js]: do we need it to be commented?

**Review by sarthaktexas — DISMISSED** [review]

File: components/layout.js (hunk 1 above)
Comment (exu3) [reply components/layout.js]: oh oops i was trying different things and i forgot to delete it

**Comment by exu3** [comment]
for the author thing, I couldn't figure out a way to make it look good with 2 authors but i'll play around with it

**Comment by sarthaktexas** [comment]
i'll add a design to the figma

**Review by zaida04 — APPROVED** [review]

**Review by sarthaktexas — APPROVED** [review]

**Pull request merged**